packages/autocomplete-preset-algolia/src/search/fetchAlgoliaResults.ts

```diff
@@ -60,6 +60,14 @@ export function fetchAlgoliaResults<TRecord>({
       })
     )
     .then((response) => {
-      return response.results;
+      return response.results.map((result) => ({
+        ...result,
+        hits: result.hits.map((hit) => ({
+          ...hit,
+          __autocomplete_algoliaResultsMetadata: {
+            analytics: (result.renderingContent as any)?.analytics || true,
```

Comment:
This is defaulted to `true` for the purpose of the proof of concept but will be left undefined otherwise.